corehq/apps/users/forms.py

```diff
@@ -1385,18 +1415,29 @@ def __init__(self, *args, **kwargs):
             ),
             crispy.Field("search_string", data_bind="value: search_string"),
         ]
+        user_type_title = _('Users')
+
         if self.user_type == MOBILE_USER_TYPE:
+            user_type_title = _('Mobile Workers')
             fields += [
                 crispy.Div(
-                    crispy.Field("location_id", data_bind="value: location_id"),
+                    crispy.Field("location_id",),
                     data_bind="slideVisible: !isCrossDomain()",
                 ),
+                crispy.Div(
+                    crispy.Field(
+                        "selected_location_only",
+                        data_bind="checked: selected_location_only"
+                    ),
+                    data_bind="slideVisible: !isCrossDomain() && location_id",
+                ),
+                crispy.Field("user_active_status",),
                 crispy.Field("columns", data_bind="value: columns"),
             ]
 
         self.helper.layout = crispy.Layout(
             crispy.Fieldset(
-                _("Filter and Download Users"),
+                _('Filter and Download {user_type_title}').format(user_type_title=user_type_title),
```

Comment:
This is going to look weird in languages that differentiate between the nominative (subject) and accusative (object) case. Translators won't know that "Users" and "Mobile Workers" above need to be in the accusative.

Better to translate the whole string:
```python
    fieldset_label = _('Filter and download users')
    if self.user_type == MOBILE_USER_TYPE:
        fieldset_label = _('Filter and download mobile workers')
        # ...
            crispy.FieldSet(
                fieldset_label, 
                # ...
```

Reply:
@kaapstorm Jeepers, good catch!